plugin/evm/header/block_gas_cost.go

```diff
@@ -27,7 +27,10 @@ func BlockGasCost(
 	config *params.ChainConfig,
 	parent *types.Header,
 	timestamp uint64,
-) uint64 {
+) *big.Int {
```

Comment:
Can we document this function like `BaseFee`?

```
// Prior to AP4, the returned block gas cost will be nil.
```

Reply:
done